pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -69,6 +73,35 @@ def get_metadata_schema(source_data: Dict[str, dict]) -> Dict[str, dict]:
     return json.loads(json.dumps(dict(results=metadata, schema=schema), cls=NWBMetaDataEncoder))
 
 
+class objectview(object):
+    def __init__(self, d):
+        self.__dict__ = d
+
+
+def validate_metadata(parent: dict, function: str) -> dict:
+    """
+    Function used to validate data using an arbitrary NWB Inspector function
+    """
+    fn = nwbinspector.__dict__.get(function)
+    if fn is None:
+        raise ValueError(f"Function {function} not found in nwbinspector")
+
+    if "subject" in function and "subject_exists" not in function:
+        cls = Subject
```

Comment:
```suggestion
        nwb_class = Subject
```
`cls` is a reserved word; not that you can't technically use it here, just not conventional